docker/pythonpath_dev/superset_config.py

```diff
@@ -99,6 +99,7 @@ class CeleryConfig:
 WEBDRIVER_BASEURL_USER_FRIENDLY = WEBDRIVER_BASEURL
 
 SQLLAB_CTAS_NO_LIMIT = True
+SLACK_API_TOKEN = os.getenv("SLACK_API_TOKEN")
```

Comment:
It could be nice to have a gitignored `superset_config_local.py` (similar to `.env-local` above)